dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/task/PythonCommandExecutor.java

```diff
@@ -118,7 +118,7 @@ protected String commandInterpreter() {
         if (StringUtils.isEmpty(pythonHome)){
             return PYTHON;
         }
-        return pythonHome;
+        return pythonHome + "/bin/python";
```

Comment:
By the way, could you modify the incorrect java docs of `private static String getPythonHome(String envPath)` and modify the `$PYTHON_HOME` to `$PYTHON_HOME/bin` in `export PATH`

Reply:
ok